assets/src/css/frontend/give-frontend.scss

```diff
@@ -49,7 +49,7 @@
 }
 
 .give-grid-ie-utility {
-	margin: 0 -12px;
+	margin: 0 -12pxz
```

Comment:
Typo?

Reply:
Definitely a typo. I'm fixing it right now.

Reply:
> Did you go over the referer idea with Ravinder?

I missed his message the other day and haven't had a chance to reconnect.  Perhaps I should add him as a reviewer? I did notice that the referrer is wrapped in logic that checks the give-receipt-session and whether the user is logged in which seems pretty secure.

Reply:
Yeah, connect with him and make him the reviewer. He'll help get this across the finish line! 💪